tests/functional/spec/components/checkbox/checkbox_visible_answers.spec.js

```diff
@@ -0,0 +1,44 @@
+const CheckboxVisibleTruePage = require('../../../generated_pages/checkbox_detail_answer_visible/checkbox-visible-true.page.js');
+const CheckboxVisibleFalsePage = require('../../../generated_pages/checkbox_detail_answer_visible/checkbox-visible-false.page.js');
+const CheckboxVisibleNonePage = require('../../../generated_pages/checkbox_detail_answer_visible/checkbox-visible-none.page.js');
+const MutuallyExclusivePage = require('../../../generated_pages/checkbox_detail_answer_visible/mutually-exclusive.page.js');
+
+
+describe('Given I start a checkbox survey with a write-in option', function() {
```

Comment:
Shouldn't the focus of the test be detail_answer and visibility not write-in and label.
I would go with something like 
"Given the checkbox detail_answer questionnaire, when a checkbox has a detail_answer with visible set to true then the detail_answer write_in should be shown"

Reply:
Added your suggestion, thanks👍